pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -362,6 +362,49 @@ func fixLibvirtCrcNetworkAvailable() error {
 	return nil
 }
 
+func removeLibvirtCrcNetwork() error {
+	logging.Debug("Removing libvirt 'crc' network")
+	_, _, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-info", libvirt.DefaultNetwork)
+	if err == nil {
```

Comment:
@cfergeau  you already answered, if the `crc` network not present their is an error which need to be ignored. I feel better using `err ==nil` instead catching it and then return `nil` :( 
```
if err != nil {
    // not sure why we ignore the error
    return nil
}
```

Reply:
But why do we want to ignore missing `crc` network? In my opinion it's better to use standard go error handling, and add an explanation, rather than using a less usual construct, where people will probably not notice that we intentionnally ignore the error.

Reply:
@cfergeau To me as per go standard if there is an error then return it and if we are just returning `nil` then it just doesn't seems right but if you say we should have a logging debug info which says that there is no crc network then I might go with that something like?
```
if err != nil {
    logging.Debug("No crc network is found for libvirt") 
    return nil
}
```

Reply:
Not even debug, just a comment I guess for future readers of the code.
Per go standard, we need to handle the error after the call. "handle" does not necessarily mean to return it, but make a decision about it. We can decide to ignore it and return nil if we know the rest of the code will be able to deal with it

Reply:
Using `if err == nil` can quickly get ugly if you need to do it several times
```
err := foo()
if err == nil {
    err = bar()
    if err == nil {
        err = foobar()
```